independent-publisher-connectors/Google BigQuery/apiProperties.json

```diff
@@ -0,0 +1,9 @@
+{
+    "properties": {
+      "connectionParameters": {},
+      "iconBrandColor": "#007ee5",
+      "capabilities": [],
+      "publisher": "Ashwani Kumar",
+      "stackOwner": "Ashwani Kumar"

```

Comment:
@ashwanidv100 Please change to API Owners.
"stackOwner": "Name of company or organization that connector is connecting to." in your case should be Google

Reply:
Ohh.. Will change this up.

Reply:
Updated to 'Google'